build.sbt

```diff
@@ -95,4 +95,11 @@ lazy val tests = project
       "org.scalatest" %% "scalatest" % "3.0.5"
     )
   )
-  .dependsOn(bsp4s, bsp4j)
+  .dependsOn(bsp4s, bsp4j, mockServer)
+
+lazy val mockServer = project
```

Comment:
Can this be moved into the `tests` module?

Reply:
Or is the intention is to publish this module eventually?

Reply:
The intention is to publish it to have a stand-alone testing app. That's my motivating use-case actually.

Reply:
Sounds good, conventionally those kinds of modules are called "testkit". 

Out of curiosity, how come you decided to use bsp4s instead of bsp4j?

Reply:
Not really a testkit at this point. But maybe it will be.
bsp4s just seemed a bit nicer and it's already there so why not?